x/ibc/03-connection/client/cli/tx.go

```diff
@@ -0,0 +1,223 @@
+package cli
+
+import (
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store/state"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
+	"github.com/cosmos/cosmos-sdk/x/ibc"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client"
+	"github.com/cosmos/cosmos-sdk/x/ibc/03-connection"
+	"github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
+)
+
+/*
+func GetTxCmd(storeKey string, cdc *codec.Codec) *cobra.Command {
+
+}
+*/
+const (
+	FlagNode1 = "node1"
+	FlagNode2 = "node2"
+	FlagFrom1 = "from1"
+	FlagFrom2 = "from2"
+)
+
+func handshake(ctx context.CLIContext, cdc *codec.Codec, storeKey string, prefix []byte, connid string) (connection.HandshakeObject, error) {
+	base := state.NewMapping(sdk.NewKVStoreKey(storeKey), cdc, prefix)
+	climan := client.NewManager(base)
+	man := connection.NewHandshaker(connection.NewManager(base, climan))
+	return man.CLIQuery(ctx, connid)
+}
+
+func lastheight(ctx context.CLIContext) (uint64, error) {
+	node, err := ctx.GetNode()
+	if err != nil {
+		return 0, err
+	}
+
+	info, err := node.ABCIInfo()
+	if err != nil {
+		return 0, err
+	}
+
+	return uint64(info.Response.LastBlockHeight), nil
+}
+
+func GetCmdConnectionHandshake(storeKey string, cdc *codec.Codec) *cobra.Command {
```

Comment:
I think I prefer having the individual handshake steps as commands and then using a shell script to chain them, but that's up to the SDK team.